models/articles.model.js

```diff
@@ -31,20 +38,18 @@ exports.selectArticles = () => {
     .then(({ rows }) => {
       rows.forEach((article) => {
         article.comment_count = Number(article.comment_count);
-      })
+      });
       return rows;
     });
-}
+};
+
+exports.checkArticleExists = (id) => {
+  return db
+    .query(`SELECT * FROM articles WHERE article_id=$1`, [id])
+    .then(({ rows: article }) => {
+      if (article.length === 0) {
+        return Promise.reject({ status: 404, message: "article not found" });
+      }
+    });
+};
```

Comment:
Really nice way of doing it, well done :) I like that it's in it's own function instead of Raw sql